tutorials/demo.rst

```diff
@@ -74,7 +75,7 @@ You can `Flash the firmware via web UI
 or via `other means available on OpenWrt
 <https://openwrt.org/docs/guide-user/installation/generic.flashing>`_.
 
-Make sure not to keep settings
+Make sure not to keep settings as shown in
```

Comment:
I am also not convinced of this one.

What about something like:

```suggestion
Make sure not to keep settings: supply the ....
```

We could remove the rounded parenthesis and add a colon before the clarification.